spec.html

```diff
@@ -40073,9 +40086,13 @@ <h1>Promise.allSettled ( _iterable_ )</h1>
         <emu-alg>
           1. Let _C_ be the *this* value.
           1. Let _promiseCapability_ be ? NewPromiseCapability(_C_).
+          1. Let _promiseResolve_ be GetPromiseResolve(_promiseCapability_).
+          1. If _promiseResolve_ is an abrupt completion, then
+            1. IfAbruptRejectPromise(_promiseResolve_, _promiseCapability_).
+            1. Return Completion(_promiseResolve_).
```

Comment:
```suggestion
            1. Return _promiseResolve_.
```
doesn't this mean the promise is rejected, but the function also throws? shouldn't it not throw in this case?

Reply:
It worked because IfAbruptRejectPromise is a macro rather than an operation. It's not revenant anymore because I changed the entire if-branch to be a single IfAbruptRejectPromise.